### ISSUE TITLE
feedvalidator.exe error: TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#55]
```
I am attempting to validate my transit feed using the latest version of
these tools on XP Pro.

Here is the output I get when attempting to validate a feed:

C:\Documents and Settings\Ryan\Desktop\dist>feedvalidator.exe
Enter Feed Location: 20070622.zip
validating 20070622.zip
Traceback (most recent call last):
  File "feedvalidator.py", line 326, in <module>
TypeError: 'NoneType' object is not iterable

The ZIP file is full of txt files as per the published guide. I have also
tried running this validator on a directory of these files, with the exact
same result.

Any help would be greatly appreciated! 
```

Original issue reported on code.google.com by `rypal...@gmail.com` on 30 Jun 2008 at 2:20

**Comment by GoogleCodeExporter** [comment]
```
Thank you for reporting the crash even though the crash handler failed. Could 
you
send your zip file to tom.brown.code@gmail.com and I'll work out what happened 
and
fix it and the crash handler in the next release.
```

Original comment by `tom.brow...@gmail.com` on 30 Jun 2008 at 6:50

**Comment by GoogleCodeExporter** [comment]
```
Ryan sent me the zip. The first crash complained about
         if (self.route_short_name and
 -->         (self.route_long_name.strip().lower().startswith(
                 self.route_short_name.strip().lower() + ' ') or

AttributeError: 'NoneType' object has no attribute 'strip'

which looks to be exactly the same as issue 54 (look there for work around).


The second crash, in the crash handler, might have something to do with py2exe. 
I
need to dig into that more.
```

Original comment by `tom.brow...@gmail.com` on 30 Jun 2008 at 5:52

**Comment by GoogleCodeExporter** [comment]
```
I'm not sure if I understand the workaround as it applies to the data I emailed 
you,
as my data did contain a column (completely populated as well) for 
'route_long_name'.
In any case, I will take a look at the next release of the validator and 
hopefully it
will work!
```

Original comment by `rypal...@gmail.com` on 30 Jun 2008 at 6:24

**Comment by GoogleCodeExporter** [comment]
```
Good point Ryan, I didn't notice that. After I removed the bad utf8 in
route_long_name the validator stopped crashing. Bad utf8 generally doesn't 
cause a
crash but issue 54 let it slip by.
```

Original comment by `tom.brow...@gmail.com` on 30 Jun 2008 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r788, to be released in 1.1.5
```

Original comment by `tom.brow...@gmail.com` on 8 Jul 2008 at 6:26
- Changed state: **Fixed**
- Added labels: **Language-Python**
